antispam_bee.php

```diff
@@ -119,7 +119,13 @@ public static function init() {
 			)
 		);
 
-		if ( ( defined( 'DOING_AJAX' ) && DOING_AJAX ) || ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) ) {
+		$disallow_ajax = apply_filters( 'antispam_bee_disallow_ajax_calls', true );
```

Comment:
I prefer using the positive phrasing: `$allow_ajax = apply_filters( 'antispam_bee_allow_ajax_calls', false );`

It might be confusing to use `true` to disable something.

Reply:
It was easier to make it this way, but I get your point.

My second thought was, "true" for default seems better than false. But I'm fine with changing the logic.

Reply:
Also valid... I approved it, because it is really a personal thing...